plugins/modules/system/alternatives.py

```diff
@@ -77,6 +77,7 @@
         description:
           - The path to the symbolic link that should point to the real subcommand executable.
         type: path
+        required: true
```

Comment:
When `link` wasn't specified, the module generated an invalid argument for `alternatives` and failed. So this change shouldn't break anything that was working.